pom.xml

```diff
@@ -15,8 +14,13 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+        <commons.version>1.10</commons.version>
+        <httpcomponents.version>4.5.1</httpcomponents.version>
+        <script-security.version>1.17</script-security.version>
+        <workflow.version>1.13</workflow.version>
```

Comment:
This Pipeline release requires Jenkins 1.609.1 or higher, so you would either need to specify `jenkins.version`, or stick with the parent POM’s current default of 1.580.1 and depend on Pipeline 1.4.2.